data/forms/warning-note-review.tsx

```diff
@@ -41,7 +48,14 @@ const formSteps: FormStep[] = [
         component: 'DateInput',
         name: 'discussedWithManagerDate',
         label: 'Date discussed with manager',
-        rules: { required: true },
+        rules: {
+          required: true,
+          validate: {
+            notInFuture: (value) =>
+              new Date(value).getTime() <= new Date().getTime() ||
+              'Date discussed with manager cannot be set in the future',
```

Comment:
try "date discussed with manager can't be in the future"

Reply:
Done in 1cbae1e